R/derive_date_vars.R

```diff
@@ -699,7 +699,7 @@ derive_vars_dtm <- function(dataset,
     date_imputation = date_imputation,
     time_imputation = time_imputation,
     min_dates = lapply(min_dates, eval_tidy, data = mask),
-    max_dates = lapply(min_dates, eval_tidy, data = mask)
+    max_dates = lapply(max_dates, eval_tidy, data = mask)
```

Comment:
Can you please check if the same error is made in `derive_vars_dt()`?

Reply:
it looks fine for derive_vars_dt()